internal/aez/photometers.go

```diff
@@ -0,0 +1,78 @@
+package aez
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+	"reflect"
+	"time"
+
+	"github.com/innosat-mats/rac-extract-payload/internal/ccsds"
+)
+
+// PMData data from photometers
+type PMData struct {
+	EXPTS    uint32 // Exposure start time, seconds (CUC time format)
+	EXPTSS   uint16 // Exposure start time, subseconds (CUC time format)
+	PM1A     uint32 // Photometer 1, thermistor input A sum
+	PM1ACNTR uint32 // Photometer 1, thermistor input A counter
+	PM1B     uint32 // Photometer 1, thermistor input B sum
+	PM1BCNTR uint32 // Photometer 1, thermistor input B counter
+	PM1S     uint32 // Photometer 1, photo diode input SIG sum
+	PM1SCNTR uint32 // Photometer 1, photo diode input SIG counter
+	PM2A     uint32 // Photometer 2, thermistor input A sum
+	PM2ACNTR uint32 // Photometer 2, thermistor input A counter
+	PM2B     uint32 // Photometer 2, thermistor input B sum
+	PM2BCNTR uint32 // Photometer 2, thermistor input B counter
+	PM2S     uint32 // Photometer 2, photo diode input SIG sum
+	PM2SCNTR uint32 // Photometer 2, photo diode input SIG counter
+}
+
+// Read PM
+func (pm *PMData) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.BigEndian, pm)
```

Comment:
Perhaps better to start reading `binary.LittleEndian` already even if we have an issue for it?